src/simcore_service_deployment_agent/cmd_utils.py

```diff
@@ -6,12 +6,12 @@
 log = logging.getLogger(__name__)
 
 
-async def run_cmd_line(cmd):
+async def run_cmd_line(cmd: str) -> str:
     proc = await asyncio.create_subprocess_shell(
         cmd, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE
     )
```

Comment:
I think we should use ```asyncio.create_subprocess_exec``` it seems less prone to security issues and also takes parameters as a list of strings